codeql_ruby/cwe-918/ServerSideRequestForgery.rb

```
@@ -0,0 +1,41 @@
+        # BAD - user can control the entire URL of the request
+        users_service_domain = params[:users_service_domain]
+        response = Excon.post("#{users_service_domain}/logins", body: {user_id: user}).body
+        token = JSON.parse(response)["token"]
+# {/fact}
+
+# {fact rule=cross-site-request-forgery@v1.0 defects=1}
+
+        # BAD - user can control the entire URL for the request using Faraday library
+        conn = Faraday.new(url: params[:url])
+        resp = conn.post
+        token = JSON.parse(resp)["token"]
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `token`.

(Lint/UselessAssignment)